conf/messages

```diff
@@ -1,6 +1,8 @@
 # https://www.playframework.com/documentation/latest/ScalaI18N
 
 main.header=Transfer Digital Records
+main.banner=beta
+main.banner.info=This is a new service – your feedback will help us to improve it.
```

Comment:
I think you normally want to include a feedback link, but maybe that's OK for private beta the users will already be in direct contact with us? Is Kirsten happy with using this wording without a link?

Reply:
I think it should be okay not have a link and add it later if I make a note. I will check with Kirsten though!